src/constants/margins.js

```diff
@@ -0,0 +1,7 @@
+export const bookContainer = { marginHorizontal: 20, marginBottom: 10 };
+export const bookImage = { marginLeft: 30, marginRight: 20 };
+export const bookDetailContainer = { marginHorizontal: 20, marginTop: 20, marginBottom: 10 };
+export const bookDetailImage = { marginLeft: 30, marginRight: 20 };
+export const bookDetailAddToWishList = { marginBottom: 10 };
```

Comment:
we just want to add constant values, not objects. Also use common names like `MARGIN_MEDIUM = 20`